demos/demos-v2.yaml

```diff
@@ -230,7 +230,7 @@ demos:
       pvc: 16Gi
   jupyterhub-keycloak:
     description: Demo showing jupyterhub notebooks secured with keycloak
-    documentation: https://docs.stackable.tech/stackablectl/stable/demos/jupyterhub-keycloak.html
+    documentation: https://docs.stackable.tech/home/stable/demos/jupyterhub-keycloak
```

Comment:
Just FYI that this is currently a 404, however it's available at https://docs.stackable.tech/home/nightly/demos/jupyterhub-keycloak/ an will be in the stable docs once SDP 25.3 is released